### ISSUE TITLE
Support Telegram "reactions" (at least read-only)

### ISSUE THREAD

**Issue by salutis** [zevlg/telega.el#358]
## Telega Setup
*OS*:  macOS 12.1
*Emacs*: GNU Emacs 29.0.50 (build 16, x86_64-apple-darwin21.2.0, NS appkit-2113.20 Version 12.1 (Build 21C52))
*Telega*: telega v0.8.11 (TDLib v1.8.1) (telega-server v0.7.13)
*MELPA*: 20220319.955

## Steps to Reproduce
1. Add a "reaction" to some message using the official Telegram client.
2. Open the chat with the reaction in Telega.

## Current Behavior
Telega does not show the reaction.

## Possible Solution
The simplest, read-only support for "reactions" to avoid missing social cues from people.

**Comment by zevlg** [comment]
Yes, this is very important feature to support in telega.  It is currently in the Work In Progress state

Support for reactions will be added soon

**Comment by salutis** [comment]
@zevlg Super!  Thank you.

**Comment by salutis** [comment]
Wohoo!

<img width="847" alt="Screen Shot 2022-05-14 at 07 49 34" src="https://user-images.githubusercontent.com/1158274/168412665-a8b56c78-a053-4261-b059-e2b0896528d0.png">

P.S. I have no idea how to select a reaction with the keyboard. 😁 

THANK YOU!

**Comment by salutis** [comment]
> P.S. I have no idea how to select a reaction with the keyboard.

I figured it out.  Turns out, Telega uses Ido completing read function instead of the default one (that the user prefers).  Fixed by:

```elisp
(setq telega-completing-read-function completing-read-function)
```

For Vertico users, consider enabling `vertico-multiform-mode` and setting:

```elisp
(add-to-list 'vertico-multiform-commands '(telega-msg-set-reaction flat))
```

Then, `C-n`/`C-p` works like everywhere and the list shows horizontally, like with Ido:

<img width="569" alt="Screen Shot 2022-05-14 at 08 05 48" src="https://user-images.githubusercontent.com/1158274/168413123-fe04fd5c-b02a-470e-95ff-6abe11a5dce9.png">

(I would still prefer a vertical list, but Telega does not describe the reactions with Unicode character names. 😞)
